fuel-crypto/Cargo.toml

```diff
@@ -22,16 +22,10 @@ serde = { version = "1.0", default-features = false, features = ["derive"], opti
 sha2 = { version = "0.10", default-features = false }
 zeroize = { version = "1.5", features = ["derive"] }
 
-# Note that, while we don't use this dependency directly, we must specify it in
-# order to enable the transitive dependency's "js" feature which is required
-# for supporting the wasm32-unknown-unknown target.
-[target.'cfg(target_arch = "wasm32")'.dependencies]
-getrandom = { version = "0.2.8", default-features = false, features = ["js"] }
-
 [dev-dependencies]
 bincode = { version = "1.3", default-features = false }
 criterion = "0.3"
-fuel-crypto = { path = ".", default-features = false, features = ["random"] }
+fuel-crypto = { path = ".", default-features = false, features = ["random", "serde"] }
```

Comment:
should this be only enabled via the serde feature flag?

Reply:
Remove it=)